app/viewtils/PenaltyTypeHelper.scala

```diff
@@ -30,6 +30,8 @@ object PenaltyTypeHelper {
       case (_, true) => messages("penaltyType.latePayment.multiple")
       case (PenaltyTypeEnum.Late_Submission, _) => messages("penaltyType.lateSubmission")
       case (PenaltyTypeEnum.Late_Payment | PenaltyTypeEnum.Additional, _) => messages("penaltyType.latePayment")
+      case _ => throw new MatchError(s"[PenaltyTypeHelper][convertPenaltyTypeToContentString] - unknown penalty type ${userRequest.answers.getAnswer[String](SessionKeys.doYouWantToAppealBothPenalties).contains("yes")
+        (penaltyType, isAppealingMultiplePenalties)}")
```

Comment:
I'm not sure what this is doing, but probably best to change this to:
```suggestion
      case _ => throw new MatchError(s"[PenaltyTypeHelper][convertPenaltyTypeToContentString] - unknown penalty type $penaltyType}")
```